### ISSUE TITLE
[BUG]: pcmToWaveBuffer run wrong

### ISSUE THREAD

**Issue by quyetvu95** [Canardoux/flutter_sound#1042]
## Flutter Sound Version : 9.4.17

When I using function pcmToWaveBuffer convert pcmToWaveBuffer and save in File (wav) . File have only 1 second duration.

**Comment by Larpoux** [comment]
No, if you want WAV record, just specify this codec during the recording. No need to use RAW PCM.

**Comment by quyetvu95** [comment]
I using PCM for recordToStream and send data wav to Server via Socket

**Comment by Larpoux** [comment]
You can record a WAV record and call waveToPCMBuffer() (and not pcmToWaveBuffer(), of course).

**Comment by Larpoux** [comment]
I suggest that you use first, files and not buffers, so that you will be able to look to them int the debugger.
And if everything OK, you may use buffers instead of files.
